main.lua

```diff
@@ -259,6 +260,13 @@ function karma_OnEvent(self, event, ...)
             loadMemberKarma();
             updateRaidList();
         end
+		if event == "CHAT_MSG_RAID_WARNING" then
+			local msg, author, language, channel = ...
+			if string.find(msg, "Please roll on") then
+				frameHidden = false;
+				karmaFrame:Show();

```

Comment:
The `kl show` command also does `updateRaidList(true)` - I don't remember why, but I assume so that what DOES show on the screen when the window un-hides is up to date.

Reply:
Yeah that's how I interpreted it. Updated any instances of Show() to that function instead

Reply:
I totally read this wrong, but fixed the problem in the latest commit